### ISSUE TITLE
Add Timestop

### ISSUE THREAD

**Issue by etra0** [etra0/botw-freecam#8]

**Comment by railgunpoi** [comment]
Hey nice mod! was wondering could you also add HUD removal? I know there's a mod that can do this but would be much more convenient if you can enable/disable with one key.

**Comment by etra0** [comment]
> 
> 
> Hey nice mod! was wondering could you also add HUD removal? I know there's a mod that can do this but would be much more convenient if you can enable/disable with one key.

@railgunpoi I don't think this will be implemented. The HUD remover from the graphic pack is already good and reimplementing it would be time consuming and the free time is limited. Sorry.
